### ISSUE TITLE
.screenshots-logo - Element is in tab order and does not have accessible text

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-services/screenshots#5017]
![image](https://user-images.githubusercontent.com/18633586/46627718-8eb92a80-caf8-11e8-9d5c-ab1e4f40c17b.png)

[detailed info on link-name rule](https://dequeuniversity.com/rules/axe/3.1/link-name?application=axeAPI)
```
Rule Violated:
link-name - Ensures links have discernible text
	URL: https://dequeuniversity.com/rules/axe/3.1/link-name?application=axeAPI
	Impact Level: serious
	Tags: cat.name-role-value wcag2a wcag412 wcag244 section508 section508.22.a
	Elements Affected:
	1) Target: .screenshots-logo
		Element does not have text that is visible to screen readers
		aria-label attribute does not exist or is empty
		aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
		Element's default semantics were not overridden with role="presentation"
		Element's default semantics were not overridden with role="none"
		Element is in tab order and does not have accessible text
```

**Comment by 123survesh** [comment]
Hi, I would like to help with this issue. 

The issue is that the above shown logo is accessible by tab key and it has no readable content that the screen readers can use right?

I found this while I was looking into how screen readers work:

>  Screen readers will read the alternative text of images, if alt text is present. JAWS precedes the alternative text with the word “graphic.” If the image is a link, JAWS precedes the alternative text with “graphic link." - https://webaim.org/techniques/screenreader/#how

The logo is not displayed as an image with <img> tag, so alt-text can't be used. It's a background image of  an empty link pointing to '#'.

So any idea on how to solve this issue?

**Comment by punamdahiya** [comment]
@123survesh tested this issue today using NVDA and Voice over screen reader and it seems few of these accessibility issues are resolved with the latest header UI in development. 

@kimberlythegeek On [dev server](https://screenshots.dev.mozaws.net/), screenshots logo has a [title Screenshots](https://github.com/mozilla-services/screenshots/blob/master/server/src/header.js#L9) home and is identified as 'link screenshots home' by the screen reader and shows up in correct tab order.  Please update if its safe to mark this issue resolved. Thanks!

**Comment by punamdahiya** [comment]
Analyzing latest screenshots UI in dev using [axe extension](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/), , marking the issue as resolved
